scripts/uberenv/packages/raja/package.py

```diff
@@ -287,8 +288,13 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
                                         hip_root))
             cfg.write(cmake_cache_entry("HIP_CLANG_PATH",
                                         rocm_root + '/llvm/bin'))
-            cfg.write(cmake_cache_entry("HIP_HIPCC_FLAGS",
-                                        '--amdgpu-target=gfx906'))
+
+            hipcc_flags = ['--amdgpu-target=gfx906']
+            if "+desul" in spec:
+                hipcc_flags.append('-std=c++14')
+            
+            cfg.write(cmake_cache_entry("HIP_HIPCC_FLAGS", ';'.join(hipcc_flags)))
```

Comment:
@kab163 Is this change what you had in mind?

Reply:
Sure! It probably doesn't make too much difference now when there's only 2 flags that could be added, but if we needed to add a bunch of flags in the future, this approach may look better.